Header header
float32[50] x
float32[50] y
float32[50] theta
float32[50] s
float32[50] height
float32[50] normal_x
float32[50] normal_y
float32[50] normal_z

